code/game/machinery/gulag_teleporter.dm

```diff
@@ -175,7 +175,7 @@ The console is located at computer/gulag_teleporter.dm
 	use_energy(active_power_usage)
 
 /obj/item/circuitboard/machine/gulag_teleporter
-	name = "labor camp teleporter (Machine Board)"
+	name = "labor camp teleporter"
```

Comment:
shouldn't this be `Labor Camp Teleporter` now like the other subtypes, they all seem to use Spaced Camel Case